codebuild_ECR_push/code_build_ecr_container.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_codebuild_project" "code_pipeline_ecr_container" {
+  name        = "code-pipeline-ecr-container"
```

Comment:
Same as the TF one. I think we need to pass in the pipeline name, environment and maybe the container name so that we get things named specifically in the console.